### PR TITLE
Establish Test Blockchain for CI - Closes #372

### LINKED ISSUE #372: Establish test blockchain for CI
### Description

Current test suite does is missing blocks and transaction in the test blockchain.

Make a snapshot of Lisk Core blockchain and integrate it with out CI process.

### Motivation

- Stable and reliable test supported by automation are crucial for quality development process

### Acceptance Criteria

- Blockchain snapshot is pushed to the branch
- Blockchain snapshot is being used in CI

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #372 

### How was it solved?
- [x] Run local `devnet` blockchain and post transactions
- [x] Export the `blockchain` and `forgers-info` from the local node
- [x] Copy the snapshots within the `lisk-core` image
- [x] Add `entrypoint.sh` script to import the snapshots and start the `lisk-core` container (node)
- [x] Enable RPC integration tests for v2 transactions
- [x] Fix source mapping for v2 transactions in gateway

### How was it tested?
Locally + Jenkins

### CODE REVIEW

File: docker/lisk-core/docker-compose.core.yml [hunk 1]
```diff
@@ -2,13 +2,16 @@ version: '3'
...
+      - ./lisk-core/snapshots/:/tmp/snapshots
+      - ./lisk-core/entrypoint.sh:/entrypoint.sh
     ports:
       - 4000
       - 5001
     networks:
       - local_lisk_core
     restart: always
-    command: ['--network=devnet', '--api-ws', '--api-ws-port=5001', '--enable-forger-plugin', '--enable-http-api-plugin', '--http-api-plugin-port=4000', '--http-api-plugin-whitelist=172.16.0.0/12']
+    entrypoint: ['sh', '/entrypoint.sh']
```
Comment (MichalTuleja) [inline docker/lisk-core/docker-compose.core.yml:14]: Does the script also work when you restart the container?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Why are `*.tar.gz` empty files?

File: docker/lisk-core/docker-compose.core.yml (hunk 1 above)
Comment (sameersubudhi) [reply docker/lisk-core/docker-compose.core.yml:14]: I did not test this scenario. Let me verify and update.

**Comment by sameersubudhi** [comment]
> Why are `*.tar.gz` empty files?

The zipped files do contain data. They're just not visible on github.
```
~/Documents/Lisk/github/lisk-service/docker/lisk-core (372-establish-test-blockchain-for-ci)$ tar -tf snapshots/blockchain.db.tar.gz 
blockchain.db/
blockchain.db/000004.sst
blockchain.db/000007.sst
blockchain.db/000010.sst
blockchain.db/000012.log
blockchain.db/CURRENT
blockchain.db/IDENTITY
blockchain.db/LOCK
blockchain.db/MANIFEST-000011
blockchain.db/OPTIONS-000011
blockchain.db/OPTIONS-000014
~/Documents/Lisk/github/lisk-service/docker/lisk-core (372-establish-test-blockchain-for-ci)$ tar -tf snapshots/forger.db.tar.gz 
forger.db/
forger.db/000004.sst
forger.db/000007.sst
forger.db/000010.sst
forger.db/000012.log
forger.db/CURRENT
forger.db/IDENTITY
forger.db/LOCK
forger.db/MANIFEST-000011
forger.db/OPTIONS-000011
forger.db/OPTIONS-000014
~/Documents/Lisk/github/lisk-service/docker/lisk-core (372-establish-test-blockchain-for-ci)$ 
```

File: docker/lisk-core/docker-compose.core.yml (hunk 1 above)
Comment (sameersubudhi) [reply docker/lisk-core/docker-compose.core.yml:14]: Updated the script to overwrite the blockchain data if it already exists.

**Review by nagdahimanshu — APPROVED** [review]
LGTM!

**Review by MichalTuleja — APPROVED** [review]
Looks good, thanks! 👍

**Pull request merged**